### DISCUSSION TITLE
Image component with static export

### DISCUSSION THREAD

**Discussion by victor-foster** [vercel/next.js#18906] [Help] [answered]
Does the Image optimizations work with static builds? Thank you!

**Comment by warrenalphonso** [comment]
It's not loading for me. I get a 404 Not Found error in console.

**Comment by rafaelalmeidatk** [comment, marked as answer]
Hi there! The image optimization is on-demand so you need a Node.js server running to optimize the images, it does not work with the `next export` command since it gives you static files.

**Reply by victor-foster** [reply]
Thank you @rafaelalmeidatk for the information (even though it's not what I wanted to hear)!

**Reply by pavlos256** [reply]
Thanks for the answer. However, it would seem that since the sizes are predefined, the static export would be able to generate all of them.
